.github/workflows/standard_spa_pr.yml

```diff
@@ -451,7 +473,7 @@ jobs:
 
   # Kicks off tests in other repos
   run-e2e-integration-tests:
-    uses: ./.github/workflows/e2e_trigger.yml
+    uses: ./.github/workflows/e2e_trigger_local.yml
```

Comment:
is this intentional?  approved in case it is, but it seems out of place to my uneducated eyes.

Reply:
addressed